js/blocks/PitchBlocks.js

```diff
@@ -194,7 +194,14 @@ function _playPitch(args, logo, turtle, blk) {
             let scaleDegree = Math.floor(arg0 - 1) % modeLength;
             let deltaOctave, semitones, deltaSemi;
             scaleDegree += 1;
-            let ref = NOTESTEP[obj[0]] -1;
+            let ref = NOTESTEP[obj[0].substr(0,1)] -1;
+            
+            if(obj[0].substr(1) === '♭') {
```

Comment:
Can you please use the constant we defined for this symbol? Same with sharp below.

Reply:
@walterbender Fixed. 
Used it for testing then forgot to replace with constants.